### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by chinmay7016**
#78 fixes
added https://fotoforensics.com/ to image section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -377,6 +377,7 @@ These can be useful for osint and social engineering.
 - [Sogou](https://pic.sogou.com/) - Chinese technology company that offers a search engine
 - [Jimpl](https://jimpl.com/) - Online photo metadata and EXIF data viewer
 - [Same Energy](https://same.energy/) - Find beautiful images
+- [fotoforensics](https://fotoforensics.com/)-Foto-Forensic
```
Comment (edoardottt) [inline README.md:380]: This must be:
```
- [FotoForensics](https://fotoforensics.com/) - Tools and training for digital picture analysis, including error level analysis, metadata, and tutorials
```

**Review by edoardottt — CHANGES_REQUESTED** [review]

**Review by chinmay7016 — COMMENTED** [review]
Sir As you said I have update the file it would be a great pleasure if you merge this request

**Review by edoardottt — APPROVED** [review]
lgtm

**Comment by edoardottt** [comment]
> Sir As you said I have update the file it would be a great pleasure if you merge this request

please stop commenting saying what people have to do or merge the request or whatever (from this or other accounts). Try to learn how open source works or just how to be polite

**Pull request merged**